shared/middleware/files_api.rb

```diff
@@ -117,6 +140,28 @@ def allowed_file_type?(endpoint, extension)
   end
 
   #
+  # PUT /v3/(assets|sources)/<channel-id>?abuse_score=<abuse_score>
```

Comment:
Perhaps `PATCH` would be more accurate? http://restcookbook.com/HTTP%20Methods/patch/